chirp/drivers/leixen.py

```diff
@@ -961,14 +961,25 @@ class JetstreamJT270MRadio(LeixenVV898Radio):
     _file_ident = b"JET"
     _model_ident = b'LX-\x89\x85\x53'
 
+class VV898SDualBank(chirp_common.Alias):
```

Comment:
I'm actually trying to eliminate this alias behavior since we started stashing metadata in the file format, so they should just be subclasses.